ido-vertical-mode.el

```diff
@@ -87,7 +87,7 @@ so we can restore it when turning `ido-vertical-mode' off")
 (defvar ido-vertical-count-active nil
   "Used internally to track whether we're already showing the count")
 
-(defcustom ido-vertical-define-keys 'C-n-and-C-p-only
```

Comment:
ah, you make a good point about least surprise. From a blank slate, I definitely would've liked to have done it your way - change as little as possible and provide options for customization. But, now we have the dilemma of sticking to existing surprising behavior that current users expect,  or surprising existing users for the sake of ... new users, I guess ?

hmm, I will ponder a bit more. :P thanks!